src/patternfly/components/Form/styles.scss

```diff
@@ -0,0 +1,462 @@
+
+  // Input - error state variables
+  --pf-c-form__input--Right--error: var(--pf-global--spacer--sm);
+
+  // Input - background
+  --pf-c-form__input--Background: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-form__input--Background--disabled: var(--pf-global--BackgroundColor--disabled);
+
+  // Input - placeholder
+  --pf-c-form__placeholder--Color: var(--pf-global--Color--dark-200);
+
+  // Check > Label
+  --pf-c-form__check__label--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-form__check__label--PaddingRight: 0;
+  --pf-c-form__check__label--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__check__label--PaddingLeft: 0;
+  --pf-c-form__check--MarginRight--addons: var(--pf-global--spacer--sm);
+
+  // Label
+  --pf-c-form__label--Color: var(--pf-global--Color--dark-200);
+  --pf-c-form__label--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-form__label--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-form__label--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__label--MarginRight: var(--pf-global--spacer--sm);
+
+  // Label - required
+  --pf-c-form__label--Left--required: -12px;
+  --pf-c-form__label--Top--required: 2px;
+
+  // Label - disabled
+  --pf-c-form__label--Color--disabled: var(--pf-global--Color--disabled);
+
+  // Label Group
+  --pf-c-form__label-group__item--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Form check
+  --pf-c-form__check__item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-form__check--Padding: var(--pf-global--spacer--sm);
+
+  // Addons
+  --pf-c-form__check__addons--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Checkbox 
+  --pf-c-form__checkbox--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-form__checkbox--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-form__checkbox--PaddingRight: 0;
+  --pf-c-form__checkbox--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__checkbox--PaddingLeft: 0;
+
+  // Radio
+  // == Change me ==
+  --pf-c-form__radio--MarginRight: var(--pf-global--spacer--md);
+
+  // Fieldset
+  --pf-c-form__fieldset--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Select
+  --pf-c-select__arrow--BorderWidth: var(--pf-global--BorderWidth--md);
+
+  // Change these to globals
+  // States
+  --pf-c-form--Color--error: var(--pf-global--danger-color--dark);
+
+  // Helpers
+  --pf-c-form__helper-text--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-form__helper-text--FontSize: var(--pf-global--FontSize--sm);
+
+  // Textarea
+  --pf-c-form__textarea--Padding: var(--pf-global--spacer--sm);
+
+  // Actions
+  --pf-c-form__form-actions--MarginRight: var(--pf-global--spacer--md);
+
+  // Tooltip
+  --pf-c-tooltip--MarginLeft: var(--pf-global--spacer--xs);
+
+  // Inline
+  --pf-c-form--MarginRight--inline: var(--pf-global--spacer--md);
+}
+
+// Base form
+.pf-c-form {
+
+  // Form group - pf-c-form__group
+  &__group {
+
+    // Inline layout
+    &.pf-m-inline {
```

Comment:
Nesting this modifier under `&__group` is unnecessary increasing it's speficity.

Is this intentional?

Reply:
Yes, it was intentional. The assumption being that '.pf-m-inline' could be used within a form, but outside of a &__group and in the broader scope, outside of a form altogether. I think it wise to at least keep within the scope of the form styling for now (detaching from &__group).